README.md

```diff
@@ -126,20 +126,20 @@ Most of the unit tests in Logstash are written using [rspec](http://rspec.info/)
 
```

Comment:
^^ (expand) should we update the paragraph above too ?

Reply:
@jakelandis hah, not sure :D probably in another PR. Though right now that statement still seems factually correct for the most part imo.